tests/test_netbox.py

```diff
@@ -91,7 +82,7 @@ def test_vms_inventory(self, requests_mock, version):
 
     @pytest.mark.parametrize("version", ["2.8.9"])
     def test_vms_inventory_pagination(self, requests_mock, version):
```

Comment:
Would also rename for the same reason.
`test_inventory_include_vms_pagination` ?

Reply:
Done.